packages/picker/src/Picker.ts

```diff
@@ -405,13 +415,33 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                     : html`
                           <span hidden id="applied-label">${appliedLabel}</span>
                       `}
-                ${this.invalid
+                ${this.invalid && !this.pending
                     ? html`
                           <sp-icon-alert
                               class="validation-icon"
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${when(this.pending, () => {
+                    import(
+                        '@spectrum-web-components/progress-circle/sp-progress-circle.js'
+                    );
+                    return html`
+                        <span
+                            aria-hidden="true"
+                            class="visually-hidden"
+                            id="loader-label"
+                        >
+                            ${this.pendingLabel}
+                        </span>
+                        <sp-progress-circle
```

Comment:
Button uses `static=white`, does design have direction on this?

Reply:
I could not find anything around this in https://spectrum.adobe.com/page/picker/, so I implemented it the same way it is in spectrum-css. But let me ping design and ask about this 👍

Reply:
@Westbrook but why is it `static=white` in Button? In https://spectrum.adobe.com/page/button/#Pending it is not static, and in light theme the color contrast is quite low:

![Screenshot 2024-02-27 at 17 18 50](https://github.com/adobe/spectrum-web-components/assets/13311865/536fadf8-3297-4d9e-95da-66ddd56250c9)